site/docs/patterns/header-block.mdx

```diff
@@ -0,0 +1,148 @@
+5. **Close button**: optional action for the user to dismiss a component or element.  
+6. **Status icon**: visual indicator that communicates status.
+
+<Diagram
+  src="/img/patterns/header-block/header-block-anatomy.png"
+  alt="A header block anatomy."
+  border
+/>
+
+### Layout
+
+Header block can be configured to be displayed using three different sizes, each differentiated by the size of the padding and the heading type. We recommend using an appropriately sized header block, relative to the container element it is embedded within.
+
+As a rule of thumb, a large container in Salt uses padding size of `--salt-spacing-300`, a medium container uses `--salt-spacing-200`, and a small container uses `--salt-spacing-100`. For example, the Dialog component uses `--salt-spacing-300` padding between elements, therefore embedding a header block also using `--salt-spacing-300` maintains a consistent layout and ensures all elements align as expected.
+
+<Diagram
+  src="/img/patterns/header-block/header-block-layout-spacing.png"
+  alt="A header block layout."
+  border
+/>
+
+<ExampleContainer type="positive" customPillText="Best practices">
+Header block with `--salt-spacing-100` padding does not contain an accent bar. We recommend using this header block size within components where visual emphasis is not important. Consider the available space for an accent bar, relative to the size of the component.
+</ExampleContainer>
+
+## Layout variations
+
+A header block has a number of possible layout variations depending on the amount of information you want to communicate, the user workflow and the container it will sit in.
+
+### With accent bar and all text elements
+
+Display headings with an accent bar which draws attention to the heading, reinforces brand identity, and remains consistent with Salt Design System’s theming options.
+
+
+<Diagram
+  src="/img/patterns/header-block/with-accent-bar-and-text.png"
+  alt="Dialog with a header block using padding --salt-spacing-300"
+  content="Dialog with a header block using padding --salt-spacing-300"
+  border
+/>
+
+### With accent bar, heading and pre-header only
+
+The dialog example demonstrates a step in a process as introduced by the pre-header and heading. A description is not necessary as the content of the dialog is sufficiently self-explanatory for the user to navigate and complete the form field instructions.
+
+<Diagram
+  src="/img/patterns/header-block/accent-bar-heading-preheader.png"
+  alt="Dialog with a header block using heading and pre header only."
+  content="Dialog with a header block using heading and pre header only."
+  border
+/>
+
+### With accent bar, heading and description only
+
+The drawer example uses a heading accompanied by a description that provides additional information on the status of the content. A contextual overview of the content is not required in the form of a pre-header, as the content itself indicates a summary of information.
```

Comment:
The spacing (whitespace between elements) in this file seems inconsistent after this point